<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>673</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kütüphane Sistemi - Kayıt Ol</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/images/beulogo_login.png</normaloff>:/images/images/beulogo_login.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Register{
background:url(:/images/images/janko-ferlic-sfL_QOnmy00-unsplash.jpg);

}

QFrame{
background:rgba(0,0,0,0.7);
border-radius:15px;
}

QFrame:hover{
background:rgba(0,0,0,0.73);
}

QLineEdit{
font-size:25px;;
color:white;
border:none;
background:transparent;
border-bottom:1px solid white;
}

QLineEdit:hover{
background:rgba(0,0,0,0.04);
}


QPushButton{
background:rgb(83, 152, 255);
border-radius:10px;
color:#fff;
}

QPushButton:hover{
border:none;
background:rgb(68, 117, 222)
}

QCheckBox{
color:silver;
background:transparent
}
QCheckBox:hover{
color:white;
}
</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background:rgba(0,0,0,0.38);
}
QFrame{
border-radius:none;
}

QFrame:hover{
background:rgba(0,0,0,0.40);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>41</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
color:silver;

border-radius:none;
}


QPushButton:hover{
background:red;
color:white
}</string>
    </property>
    <property name="text">
     <string>✕</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimize">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>41</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
border-radius:none;
}


QPushButton:hover{
background:rgba(0, 0, 0,0.6)
}</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>3</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/images/beulogo_login.png&quot; width=&quot;21&quot; height=&quot;30&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>12</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:silver;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Kayıt Ol</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>75</y>
     <width>560</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="placeholderText">
     <string>Ad</string>
    </property>
   </widget>
   <widget class="QLabel" name="baslik">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>380</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:40px;
color:white;
background:transparent;</string>
    </property>
    <property name="text">
     <string>KÜTÜPHANE SİSTEMİ</string>
    </property>
   </widget>
   <widget class="QLabel" name="baslik_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:18px;
color:white;
background:transparent;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>BÜLENT ECEVİT ÜNİVERSİTESİ</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>144</y>
      <width>250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Soyad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>225</y>
      <width>250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="placeholderText">
     <string>Kullanıcı Adı</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>225</y>
      <width>250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Şifre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_mail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>305</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>E-Posta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="telefon_no">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>375</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Telefon No</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_createAccount">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>474</y>
      <width>400</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Hesap Oluştur</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/images/beulogo_login.png&quot; width=&quot;67&quot; height=&quot;105&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>baslik</zorder>
   <zorder>name</zorder>
   <zorder>baslik_2</zorder>
   <zorder>surname</zorder>
   <zorder>username</zorder>
   <zorder>password</zorder>
   <zorder>e_mail</zorder>
   <zorder>telefon_no</zorder>
   <zorder>btn_createAccount</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
